src/components/Skeletons/SearchRowSkeleton.tsx

```diff
@@ -5,23 +5,39 @@ import useWindowDimensions from '@hooks/useWindowDimensions';
 import CONST from '@src/CONST';
 import ItemListSkeletonView from './ItemListSkeletonView';
 
-type TableListItemSkeletonProps = {
+type SearchRowSkeletonProps = {
     shouldAnimate?: boolean;
     fixedNumItems?: number;
+    gradientOpacityEnabled?: boolean;
 };
 
-const barHeight = '10';
-const shortBarWidth = '40';
-const longBarWidth = '120';
+const barHeight = 8;
+const longBarWidth = 120;
 
-function TableListItemSkeleton({shouldAnimate = true, fixedNumItems}: TableListItemSkeletonProps) {
+// 375 is the width of the left pane
+const leftPaneWidth = 375;
```

Comment:
There must be a const or something for this

Reply:
I set video ratio to match modal with animation size. There is no example video or something I can relate to